### PR TITLE
Refer to Issue #64: Add the class for header authorization

### PR DESCRIPTION
**Pull request by robinkorkmaz1**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <maven.compiler.source>17</maven.compiler.source>
-        <maven.compiler.target>${maven.compiler.source}</maven.compiler.target>
```
Comment (kappsegla) [inline pom.xml:13]: We do not want to update pom.xml java version.

**Review by kappsegla — CHANGES_REQUESTED** [review]
Remove changes to pom.xml from pullrequest that changes java version from 17 to 16

**Review by robinkorkmaz1 — COMMENTED** [review]
Ok

**Comment by kappsegla** [comment]
> Ok
Should be possible to fix this with:
git checkout pull-request-branch

Overwrite the modified file(s) with the file in another branch:
git checkout origin/master -- pom.xml

Commit and push it to the remote:
git commit -m "removed a modified file from PR"
git push

**Review by BorvD — APPROVED** [review]

**Comment by kappsegla** [comment]
Stale implementation. Lets close it and open a new PR if the issue still seams relevant.

**Pull request closed without merging**